main.js

```diff
@@ -0,0 +1,157 @@
+// 1 //
+
+const validity = function (value) {
+  if (value === undefined || value === null) {
+    return;
+  }
+  return String(value);
+};
+
+const autoTeam = function (
+  teamName,
+  driverName,
+  yearBirth,
+  brandCar,
+  sponsor,
+  driverAcess
+) {
+  const newAutoTeam = {
+    teamName: validity(teamName),
+    driverName: validity(driverName),
+    yearBirth: validity(yearBirth),
+    brandCar: validity(brandCar),
+    sponsor: validity(sponsor),
+    driverAcess: driverAcess,
```

Comment:
I think it should be boolean, and there is no validation for the field
<img width="603" alt="image" src="https://user-images.githubusercontent.com/84010501/192340524-b62839bf-ca84-4102-8ac4-f2188a9a029f.png">